pinot-controller/src/main/resources/Readme.md

```diff
@@ -29,7 +29,7 @@ This package contains code for Pinot Controller UI.
 ```shell
 cd pinot-controller/src/main/resources
 ```
-3. Install Required Packages. Make sure you are using node v14 or more specifially v14.18.1
+3. Install Required Packages. Make sure you are using node v14 or more specifically v16.15.0
```

Comment:
Should we also modify `v14` to `v16`?

Reply:
this is the change right.

Reply:
Do we need to also update the major version?
```suggestion
3. Install Required Packages. Make sure you are using node v16 or more specifically v16.15.0
```

Reply:
Right I missed this.